crates/turborepo-lib/src/retry.rs

```diff
@@ -0,0 +1,46 @@
+use std::future::Future;
+
+use anyhow::anyhow;
+use tokio::time::sleep;
+
+const MIN_SLEEP_TIME: u64 = 2;
+const MAX_SLEEP_TIME: u64 = 10;
```

Comment:
Just for better readability
```suggestion
const MIN_SLEEP_TIME_SECS: u64 = 2;
const MAX_SLEEP_TIME_SECS: u64 = 10;
```